### PR TITLE
kafka 0.9.0.1 (new formula)

### PR DESCRIPTION
**Pull request by OndroNR**
- [X] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md) document?
- [X] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-versions/pulls) for the same formula update/change?
- [X] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [X] Does your submission pass `brew audit --strict --online <formula>` (after doing `brew install <formula>`)?
- [X] Does your submission adhere to Versions "Acceptable Formulae" [guidelines](https://github.com/Homebrew/homebrew-versions/blob/master/README.md#acceptable-formulae)?
---

### CODE REVIEW

File: kafka090.rb [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+  sha256 "db28f4d5a9327711013c26632baed8e905ce2f304df89a345f25a6dfca966c7a"
+
+  bottle do
+    cellar :any_skip_relocation
+    rebuild 1
+    sha256 "16b2d2ad87154ee65f7b7e90bf2344aa495e518fc41a000642bfbb37e0bc329b" => :sierra
+    sha256 "2874959d241121c73d914ea4d7b00055274d1bbeb4680d899e80243095c5d523" => :el_capitan
+    sha256 "16b2d2ad87154ee65f7b7e90bf2344aa495e518fc41a000642bfbb37e0bc329b" => :yosemite
+    sha256 "16b2d2ad87154ee65f7b7e90bf2344aa495e518fc41a000642bfbb37e0bc329b" => :mavericks
+  end
```
Comment (MikeMcQuaid) [inline kafka090.rb]: Remove the old bottles.

Comment (OndroNR) [inline kafka090.rb]: Done

**Comment by OndroNR** [comment]
I just started to get
```
brew audit --new-formula kafka090
Error: comparison of Version::FromURL with nil failed
Please report this bug:
  https://git.io/brew-troubleshooting
/usr/local/Homebrew/Library/Homebrew/dev-cmd/audit.rb:671:in `<'
/usr/local/Homebrew/Library/Homebrew/dev-cmd/audit.rb:671:in `audit_revision_and_version_scheme'
/usr/local/Homebrew/Library/Homebrew/dev-cmd/audit.rb:1030:in `audit'
/usr/local/Homebrew/Library/Homebrew/dev-cmd/audit.rb:72:in `block in audit'
/usr/local/Homebrew/Library/Homebrew/dev-cmd/audit.rb:68:in `each'
/usr/local/Homebrew/Library/Homebrew/dev-cmd/audit.rb:68:in `audit'
/usr/local/Homebrew/Library/Homebrew/brew.rb:94:in `<main>'
```
without changing Formula.

And also after keeping only sierra bottle, Jenkins now fails on removed macOS verions bottles.

File: kafka090.rb [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+  homepage "https://kafka.apache.org"
+  url "http://mirrors.ibiblio.org/apache/kafka/0.9.0.1/kafka_2.11-0.9.0.1.tgz"
+  mirror "https://archive.apache.org/dist/kafka/0.9.0.1/kafka_2.11-0.9.0.1.tgz"
+  sha256 "db28f4d5a9327711013c26632baed8e905ce2f304df89a345f25a6dfca966c7a"
+
+  bottle do
+    cellar :any_skip_relocation
+    rebuild 1
+    sha256 "16b2d2ad87154ee65f7b7e90bf2344aa495e518fc41a000642bfbb37e0bc329b" => :sierra
+  end
```
Comment (MikeMcQuaid) [inline kafka090.rb]: Remove the whole `bottle do` block.

File: kafka090.rb [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+  conflicts_with "kafka", :because => "Differing versions of the same formula"
+  conflicts_with "kafka080", :because => "Differing versions of the same formula"
+
+  # Related to https://issues.apache.org/jira/browse/KAFKA-2034
+  # Since Kafka does not currently set the source or target compability version inside build.gradle
+  # if you do not have Java 1.8 installed you cannot used the bottled version of Kafka
+  pour_bottle? do
+    reason "The bottle requires Java 1.8."
+    satisfy { quiet_system("/usr/libexec/java_home --version 1.8 --failfast") }
+  end
```
Comment (MikeMcQuaid) [inline kafka090.rb]: You can remove this as you're depending on Java 1.8 above.

**Comment by OndroNR** [comment]
Just made requested changes.

**Comment by MikeMcQuaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

For future reference the preferred commit message format for new formulae is `kafka090 0.9.0.1 (new formula)`. Please complete the issue template in future PRs where this format is detailed in the linked [Contributing guidelines](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md). Thanks!

**Pull request closed without merging**